website/site/content/docs/add-to-your-site.md

```diff
@@ -80,7 +80,7 @@ Configuration will be different for every site, so we'll break it down into part
 
 ### Backend
 
-Because we're using GitHub and Netlify for our hosting and authentication, backend configuration is fairly strightforward. You can start your `config.yml` file with these lines:
+Because we're using [Netlify](https://www.netlify.com) for our hosting and authentication, backend configuration is fairly strightforward. You can start your Netlify CMS `config.yml` file with these lines:
```

Comment:
Maybe this should be changed to "If we are using Netlify for our hosting", since it's not required.

Reply:
Please also fix strightforward => straightforward while you're at it :)